DESCRIPTION

```diff
@@ -1,13 +1,14 @@
 Type: Package
 Package: mmrm
 Title: Mixed Models for Repeated Measures
-Version: 0.0.0.9010
+Version: 0.0.0.9011
 Authors@R: c(
     person("Daniel", "Sabanes Bove", , "daniel.sabanes_bove@roche.com", role = c("aut", "cre")),
     person("Julia", "Dedic", , "julia.dedic@roche.com", role = "aut"),
     person("Brian Matthew", "Lang", , "brian.lang@msd.com", role = "aut"),
     person("Craig", "Gower-Page", , "craig.gower-page@roche.com", role = "aut"),
-    person("Alessandro", "Noci", , "alessandro.noci@roche.com", role = "aut")
+    person("Alessandro", "Noci", , "alessandro.noci@roche.com", role = "aut"),
+    person("Doug", "Kelkhoff", , "kelkhoff.douglas@gene.com", role = "aut")
```

Comment:
I would suggest to move your name between Brian and Craig, since Craig and Alessandro are here mostly because we copied some function from rbmi, but they are not direct authors with new contributions here